components/AudioPlayer.module.css

```diff
@@ -0,0 +1,236 @@
+.audio_player_section_container {
+	position: relative;
+	width: 100%;
+	height: 100%;
+	/* @@ updated */
+	padding: 0 3rem;
+	display: grid;
+	grid-template-columns: 100%;
+	grid-template-rows: 10% 30% 45% 15%;
+	justify-content: center;
+	align-items: center;
+	/* @@ updated */
+}
+
+.image_wrapper {
+	/* @@ updated */
+	position: relative;
+	width: 60%;
+	height: 90%;
+	/* @@ updated */
+	margin: 1rem auto;
+	box-shadow: 0 0 20px black;
+}
+
+.episode_info_section {
+	/* @@ updated */
+	height: 100%;
+	/* align-self: flex-start; */
+	align-self: center;
+	display: grid;
+	grid-template-columns: 100%;
+	grid-template-rows: auto auto;
+	gap: 1rem;
+	padding-bottom: 1.5rem;
+	/* @@ updated */
+}
+
+.episode_title {
+	text-align: center;
+	/* @@ updated */
+	margin: 1rem 0;
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+	gap: 0.5rem;
+	/* @@ updated */
+}
+
+/* @@ updated */
+.episode_title p {
+	font-weight: 600;
+	margin: 0;
+}
+/* @@ updated */
+
+.episode_details {
+	/* @@ updated */
+	position: relative;
+	padding: 0 1rem;
+	/* @@ updated */
+	overflow: scroll;
+	text-align: start;
+}
+
+.episode_details::-webkit-scrollbar {
+	display: none;
+}
+
+/* @@ updated */
+.episode_details p {
+	margin: 0;
+}
+/* @@ updated */
+
+.audioPlayer {
+	--primary: grey;
+	--secondary: white;
+	/* @@ updated */
+	text-align: center;
+	align-self: flex-start;
+}
+
+.audioPlayerPlaybutton {
+	display: flex;
+	justify-content: center;
+	gap: 20px;
+	/* @@ updated */
+	margin-bottom: 1rem;
+}
+
+.forwardBackward {
+	background: none;
+	border: none;
+	display: flex;
+	align-items: center;
+	font-family: var(--title-font);
+	font-size: 12px;
+	cursor: pointer;
+}
```

Comment:
```suggestion
position: relative;
}

.forwardBackward svg {
		position: relative;
		top: 0.2rem;
	}
```
![スクリーンショット 2021-12-13 0 05 55](https://user-images.githubusercontent.com/51708229/145774711-c2adac90-073d-4e46-bdd8-8f811b5f2edb.png)

Even though having `align-items: center`, the arrow and "30" align a bit off, so we can adjust it like this.
I haven't checked every device so it might be off again in the different devices.

Reply:
Great suggestion! This seems to work.